### PR TITLE
Add germany/hessen/ioki

### PR DESCRIPTION
**Pull request by fabrik42**
Hi there!

I would like to add my company to the registry!

I tried to follow the README as good as possible. Please let me know if I should change something.

Cheers!
Christian

### CODE REVIEW

File: src/_companies/europe/germany/hessen/ioki/index.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+stack:
+  - "rails"
+  - "postgres"
+  - "react"
+
+created_at: 2023-03-20 10:18:00
+---
+
+ioki, founded in 2018, is a startup based in Germany that works on the future of mobility. It is a subsidary of Deutsche Bahn and its mission is to empower the public transport sector. With modern technology and practices we try to shape the public transport of tomorrow.
```
Comment (CalvinWalzel) [inline src/_companies/europe/germany/hessen/ioki/index.md:24]:
```suggestion
ioki, founded in 2018, is a startup based in Germany that works on the future of mobility. It is a subsidary of Deutsche Bahn and its mission is to empower the public transport sector. With modern technology and practices they try to shape the public transport of tomorrow.
```

**Review by CalvinWalzel — COMMENTED** [review]
Hey @fabrik42, thanks for the contribution! I've added a minor comment :)

**Review by CalvinWalzel — APPROVED** [review]

**Comment by fabrik42** [comment]
Hi,
Thanks for merging!
Sorry, I fell last week and could not respond earlier.

I can also create a follow-up PR to fix the text. :)

Have a great week!

**Comment by CalvinWalzel** [comment]
No worries, I hope you're better! I've already fixed the text after merging, but if there's more info you would like to add (or you have a suggestion what other information to show for all companies), feel free to open a pull request or an issue with feedback :)

**Pull request merged**